### PR TITLE
Firefox Home New Tab

### PR DESCRIPTION
**Pull request by sarinali**
# Description

Verifies that upon changing the default new tab, the correct content is shown.

## Bugzilla bug ID

**Testrail:** https://testrail.stage.mozaws.net/index.php?/cases/view/161472
**Link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1905167

## Type of change

Please delete options that are not relevant.

- [x] New Test

# How does this resolve / make progress on that bug?

Completes.

# Screenshots / Explanations

N/A

# Comments / Concerns

Verifying about:home in this code means checking that the logo shows up. Note that on about:blank, the logo doesn't exist and those are the only two options so hence why I chose that as an indicator.

### CODE REVIEW

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]
Just one thing, but I think it's important

File: tests/preferences/test_firefox_home_new_tabs.py
[the remark this answers is not in the document]
Comment (sarinali) [reply tests/preferences/test_firefox_home_new_tabs.py:1]: This is part of the new suite, listed in the sheet. Should that be moved as well?

[the remark this answers is not in the document]
Comment (ben-c-at-moz) [reply tests/preferences/test_firefox_home_new_tabs.py:1]: Wait, ok hold on...

[the remark this answers is not in the document]
Comment (sarinali) [reply tests/preferences/test_firefox_home_new_tabs.py:1]: all good, i moved it since theyre the same suite

**Review by ben-c-at-moz — APPROVED** [review]
LGTM!

**Review by ben-c-at-moz — APPROVED** [review]
LGTM

**Pull request merged**